### ISSUE TITLE
Incorrect site behaviours, after upstream update

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/ABLE#731]
I think an upstream iForm update has broken EBMS to some extent, I am looking into it.

Two things I have noticed:

1. Website encountered an error, problem when signing up (although the registration email does arrive)
2. I tried cancelling one of my test accounts, and I got a message about curl.

**Comment by andrewvanbreda** [comment]
Fixed.
Problem is a bug is the User Registration Password module that is known but the module creator has not deployed the fix yet.
https://www.drupal.org/project/user_registrationpassword/issues/3467248

Issue seen on other projects

The fix I included is the simple
$output = $mail['result'] ?? FALSE;
change to
$output = $mail ?? FALSE;

@BirenRathod
Note: I have added the module to Custom for EBMS as per NPMS/Plant Portal.
UKBMS seems to use a different registration mode, so this is not needed.
